index.js

```diff
@@ -816,6 +964,187 @@ module.exports = class Combobo {
     return this;
   }
 
+  /**
+   * Clear all options from the Combobo. This will remove all options from the Combobo and
+   * reset the Combobo to its initial state.
+   * 
+   * @param {Object} args
+   * @param {String} [args.placeholder] The placeholder text to be added back to the Combobo.
+   * @param {Boolean} [args.usePlaceholder] If true, the placeholder option that's added back will be selected.
+   */
+  clearOptions(args = {}) {
```

Comment:
@stephymiehle  I noticed the addOptions and clearOptions methods in the recent changes. I wasn’t able to find their usages in the current implementation. If these methods were intended as improvements, would it be possible to include them in a separate PR and commit for better clarity?

Additionally, are the other functionalities tested when options are added or removed dynamically? If addOptions and clearOptions are not intended for internal use, perhaps we could consider updating the documentation to reflect their purpose.